src/plugin/range/index.js

```diff
@@ -0,0 +1,65 @@
+
+export class DayjsRange {
+  constructor(start, end) {
+    let s = start
+    let e = end
+
+    if (s > e) {
```

Comment:
Small improvement to do this as a one liner
```suggestion
const { s, e } = start > end ? { s: end, e: start } : { s: start, e: end }
```